### DISCUSSION TITLE
'sudo iwconfig':  should hci0 (Bluetooth) be listed?

### DISCUSSION THREAD

**Discussion by sudo-joe** [nzymedefense/nzyme#1148] [Questions]
Hello,

´´´
ubuntu@ubuntu:~$ sudo hciconfig
hci0:	Type: Primary  Bus: USB
	BD Address: F4:bb:cc:ee:ff:gg  ACL MTU: 1022:8  SCO MTU: 183:5
	UP RUNNING 
	RX bytes:1372 acl:0 sco:0 events:97 errors:0
	TX bytes:5792 acl:0 sco:0 commands:97 errors:0

ubuntu@ubuntu:~/Desktop$ sudo hciconfig hci0 up
ubuntu@ubuntu:~/Desktop$ sudo iwconfig
lo        no wireless extensions.

enp3s0f2  no wireless extensions.

wlp4s0    IEEE 802.11  ESSID:off/any  
          Mode:Managed  Access Point: Not-Associated   Tx-Power=16 dBm   
          Retry short limit:7   RTS thr:off   Fragment thr:off
          Encryption key:off
          Power Management:off
          
´´´

**Comment by lennartkoopmann** [comment]
No, `iwconfig` only lists WiFi devices.
